### DISCUSSION TITLE
Catcher size does not match stable.

### DISCUSSION THREAD

**Discussion by Doxxz** [ppy/osu#18870] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/67333710/175897370-9e92e029-50df-4f33-bc10-37a34a69bed3.png)
![image](https://user-images.githubusercontent.com/67333710/175897507-dcbd4c79-5ea3-43a5-9049-f8079d45fa3d.png)
Is this intended?

**Comment by peppy** [comment, marked as answer]
I think we may have adjusted the catcher to actually more correctly align to the width you can catch, but not 100% (@ekrctb may know off the top of their head).

Could you provide a link to your skin just in case it's something related to the skin?

**Reply by Doxxz** [reply]
https://www.dropbox.com/s/u3f4y7aiia6kxp2/hi.osk
Though I've tried with a few other skins and they all show the same results.

**Reply by ekrctb** [reply]
I don't know if this is intentional (it uses the same code for default skin and legacy skin). But probably it should be fixed as a bug for legacy skins because some user skins are designed for the stable's scaling in mind.

**Reply by peppy** [reply]
Tracking via https://github.com/ppy/osu/issues/18968.
